diff --git a/src/connection.c b/src/connection.c
index 6ab20ab..7c6767f 100644
--- a/src/connection.c
+++ b/src/connection.c
@@ -26,7 +26,7 @@
 
 #define _GNU_SOURCE
 
-#include "../config.h"
+#include "config.h"
 
 #include <math.h>
 #include <stdlib.h>
diff --git a/src/wayland-os.c b/src/wayland-os.c
index f00ead4..0869415 100644
--- a/src/wayland-os.c
+++ b/src/wayland-os.c
@@ -25,7 +25,7 @@
 
 #define _GNU_SOURCE
 
-#include "../config.h"
+#include "config.h"
 
 #include <sys/types.h>
 #include <sys/socket.h>
